docs/theming.md

```diff
@@ -1,113 +1,209 @@
+[standard themes](https://material.io/guidelines/style/color.html#color-themes)
+with shades that are carefully chosen for each purpose within the UI. The CSS files are
+designed so that the names are consistent between themes (so the same "purpose" will always
+have the same class name across CSS files). This makes it easy to switch themes simply by
+changing a few CSS definitions. Material Design also defines the relationship between
+the different shades, so you can calculate your own color values or, more straightforwardly, use
+an [online palette design tool](http://mcg.mbitson.com/).
+
+See the [Material Design Style page](https://material.io/guidelines/style/color.html#) for
+more information about color concepts.
 
 ## Using a pre-built theme
 
-Angular Material comes prepackaged with several pre-built theme css files. These theme files also
-include all of the styles for core (styles common to all components), so you only have to include a
-single css file for Angular Material in your app.
-
-You can include a theme file directly into your application from
-`@alfresco/adf-core/prebuilt-themes`
-
-Available pre-built themes:
-* `adf-blue-orange.css`
-* `adf-blue-purple.css`
-* `adf-cyan-orange.css`
-* `adf-cyan-purple.css`
-* `adf-green-orange.css`
-* `adf-green-purple.css`
-* `adf-indigo-pink.css`
-* `adf-pink-bluegrey.css`
-* `adf-purple-green.css`
-
-If you're using Angular CLI you can include one of the prebuilt theme in your `styles.scss` file:
-```css
-@import '~@alfresco/adf-core/prebuilt-themes/adf-blue-orange.css';
-```
-Or you can add it directly in your index.html
+ADF supplies several
+[pre-built theme files](https://github.com/Alfresco/alfresco-ng2-components/tree/master/lib/core/styles/prebuilt)
+that you can use in your app very easily.
 
-```html
-<link href="node_modules/ng2-alfresco-core/prebuilt-themes/adf-blue-orange.css" rel="stylesheet">
-```
+If you have built the app using the
+[ADF Yeoman Generator](https://github.com/alfresco/generator-ng2-alfresco-app) then you can
+change the main theme file imported into the `src/custom-style.scss` file:
 
-## Defining a custom theme
+```css
+@import '~@alfresco/adf-core/prebuilt-themes/adf-green-purple.css';
+```
 
-When you want more customization than a pre-built theme offers, you can create your own theme file. You need to include the packages only what you really use in your application.
+For example, you could change `adf-green-purple.css` to `adf-indigo-pink.css`, etc. You also
+need to comment out the color definitions below to prevent them from overriding the
+main theme from the imported file:
 
-```scss
-/*
- *  Include only packages that you are using (and core by default)
- */
-@import '~@alfresco/adf-content-services/theming';
-@import '~@alfresco/adf-process-services/theming';
-@import '~@alfresco/adf-insights/theming';
-@import '~@alfresco/adf-core/theming';
+```css
+@import '~@alfresco/adf-core/prebuilt-themes/adf-indigo-pink.css';
```

Comment:
This partial example doesn't seems to help as.it should my opinion is that the complete example of the previous version was better .